pkg/api/vminfo.go

```diff
@@ -69,6 +76,10 @@ type VmMemInfo struct {
 }
 
 type VmConfig struct {
+	// AutoMigrationEnabled indicates to the scheduler plugin that it's allowed to trigger migration
+	// for this VM. This defaults to false because otherwise we might disrupt VMs that don't have
+	// adequate networking support to preserve connections across live migration.
+	AutoMigrationEnabled bool `json:"autoMigrationEnabled"`
```

Comment:
Where do we set the default value for `AutoMigrationEnabled`?

Reply:
Default is defined by how this value is set:

https://github.com/neondatabase/autoscaling/blob/aced6c486f370d989514e7b3a6e9ddd15543ee39/pkg/api/vminfo.go#L172

That calls `hasTrueLabel` which returns true iff the label is present and equals `"true"`.